source/_integrations/tplink_omada.markdown

```diff
@@ -3,7 +3,7 @@ title: TP-Link Omada SDN Controller
 description: Instructions on integrating TP-Link Omada SDN networking devices to Home Assistant.
 ha_category:
   - Hub
-ha_release: 2023.3
+ha_release: 2023.4
```

Comment:
This should not be updated. It indicates the release the integration was introduced, which will not change :)

```suggestion
ha_release: 2023.3
```

Reply:
Yeah, that makes a lot more sense :)